src/main/java/com/exadel/project/skill/service/SkillService.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.project.skill.service;
+
+import com.exadel.project.skill.entity.Skill;
+import com.exadel.project.skill.repository.SkillRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
+public class SkillService {
+    private SkillRepository skillRepository;
+
+    public List<Skill> getAllSkills(){
+        return skillRepository.findAll();
+    }
+
+    public List<Skill> getSkillsByIds(Set<Long> ids){
+        return ids.stream()
+                .map(this::getSkillById)
```

Comment:
It's not a good way to get info from BD, If the request contains 1000 cats, then the program will make 1000 calls to the base, so it's wrong. The number of DB calls should not depend on the number of ids sent in the request.
Pls, use `in` sql operator instead, so you can get all info you needed by 1 call.

Reply:
fixed